website/homepage/src/docs/__init__.py

```diff
@@ -101,12 +101,30 @@ def find_cls(target_cls):
                 return cls
     raise ValueError("Class not found")
 
-def build(output_dir, jinja_env):
+def build(output_dir, jinja_env, gradio_wheel_url, gradio_version):
     os.makedirs(output_dir, exist_ok=True)
     template = jinja_env.get_template("docs/template.html")
-    output = template.render(docs=docs, find_cls=find_cls)
+    output = template.render(docs=docs, find_cls=find_cls, version="main", gradio_version=gradio_version, gradio_wheel_url=gradio_wheel_url)
     output_folder = os.path.join(output_dir, "docs")
     os.makedirs(output_folder)
-    output_file = os.path.join(output_folder, "index.html")
+    output_main = os.path.join(output_folder, "main")
+    os.makedirs(output_main)
+    output_file = os.path.join(output_main, "index.html")
     with open(output_file, "w") as index_html:
         index_html.write(output)
+    template = jinja_env.get_template(f"docs/v{gradio_version}_template.html")
+    output = template.render()
+    version_docs_file = os.path.join(output_folder, "index.html")
+    with open(version_docs_file, "w") as index_html:
+        index_html.write(output)
+
+def build_pip_template(version, jinja_env):
+    docs_files = os.listdir("src/docs")
+    for file in docs_files:
+        if file.endswith("_template.html") and file.startswith("v"):
+            os.remove(os.path.join("src/docs", file))
```

Comment:
We don't need to remove the old version builds right? Might be good to keep them for just-in-case even though there are not displayed on the website?

Reply:
yup you're right, might be useful! done